lib/weatherApp.js

```diff
@@ -0,0 +1,46 @@
+(function (factory) {
+    if (typeof define === 'function' && define['amd']) {
+        define(['knockout'], factory);
+    } else if (typeof require === 'function' && typeof exports === 'object' && typeof module === 'object') {
+        module.exports = factory(require('knockout'));
+    } else {
+        window.WeatherApp = factory(ko);
+    }
+} (function (ko) {
+    return function WeatherApp() {
+        'use strict';
+        let city = ko.observable("");
+        let api = "http://api.openweathermap.org/data/2.5/weather?q=";
```

Comment:
It would be nice to get rid of all of these variables and inline them in the fetch call using an ES6 template string. I think it would make the code much more readable.